generator/parseXML/ParseXML.py

```diff
@@ -184,24 +185,38 @@ def get_enum_name(self, node):
 
     @staticmethod
     def get_mapping(self, node):
+        '''
+        Used to get information from a <mapping> node
+        e.g.
+        <mapping name="DimensionDescription" package="numl"/>
+
+        :param node: the <mapping> node to interrogate
+        :return: returns [name, package] tuple
+        '''
         name = ''
-        package = ''
+        package = ''  # TODO Maybe better as None?
```

Comment:
Agreed

Reply:
Do we need to change the calling code though? (In function `parse_deviser_xml()`)

Reply:
Shouldnt have thought so !